### PR TITLE
Additional tests for ledger api test tool

### PR DESCRIPTION
**Pull request by gerolf-da**
Small fixes and more optional tests for ledger-api-test-tool.

- TransactionBackpressureIT
- DivulgenceIT
- CommandTransactionChecksHighLevelIT
- CommandTransactionChecksLowLevelIT

Contributes to #1374.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/DivulgenceIT.scala [hunk 1]
```diff
@@ -139,26 +142,38 @@ class DivulgenceIT
...
+        .runWith(Sink.head)
     } yield ()
 
   // here div1Cid is _divulged_ to Bob after it is created. the checks below
   // check that this divulgence is not visible in the ACS / flat transaction stream,
   // but that it is visible in the transaction trees.
   case class Setup(div1Cid: String, div2Cid: String)
 
-  private def createDivulgence1(ctx: LedgerContext): Future[ContractIdString] =
+  private val runSuffix: UUID = UUID.randomUUID
```
Comment (rautenrieth-da) [inline ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/DivulgenceIT.scala]: At some point, we should factor out all ID mangling into some helper class or scalatest trait. See also https://github.com/digital-asset/daml/pull/1735.

Comment (gerolf-da) [reply ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/DivulgenceIT.scala]: Yes definitely

**Comment by stefanobaghino-da** [comment]
For future referencee, please note that the `unreleased.rst` file now has to not be divided in sections, but the name of the section should be prepended in square brackets before the release note as in the following example:

    - [Ledger API] Added additional Ledger API integration tests to Ledger API Test Tool.

It will be a task of the person in charge of the release re-arrange these correctly in sections in the final release notes.

It's more laborious on release but this ensures that a release note line is not mistakenly moved across sections by an automatic merge, causing a cascade of copy-pasta-disastah. :wink:

File: ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/DivulgenceIT.scala (hunk 1 above)
Comment (gleber-da) [reply ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/DivulgenceIT.scala]: I am doing this as part of #1735

**Pull request merged**